test/integration/targets/ecs_cluster/playbooks/network_force_new_deployment.yml

```diff
@@ -0,0 +1,110 @@
+- hosts: localhost
+  connection: local
+  vars:
+     resource_prefix: 'ansible-testing-fnd'
+
+  tasks:
+    - block:
+        - name: set up aws connection info
+          set_fact:
+            aws_connection_info: &aws_connection_info
+              aws_access_key: "{{ aws_access_key }}"
+              aws_secret_key: "{{ aws_secret_key }}"
+              security_token: "{{ security_token }}"
+              region: "{{ aws_region }}"
+          no_log: True
+
+        - name: create ecs cluster
+          ecs_cluster:
+            name: "{{ resource_prefix }}"
+            state: present
+            <<: *aws_connection_info
+
+        - name: create ecs_taskdefinition
+          ecs_taskdefinition:
+            containers:
+              - name: my_container
+                image: ubuntu
+                memory: 128
+            family: "{{ resource_prefix }}"
+            state: present
+            <<: *aws_connection_info
+          register: ecs_taskdefinition_creation
+
+        - name: create ecs_service
+          ecs_service:
+            name: "{{ resource_prefix }}"
+            cluster: "{{ resource_prefix }}"
+            task_definition: "{{ resource_prefix }}"
+            desired_count: 1
+            state: present
+            <<: *aws_connection_info
+          register: ecs_service_creation
+
+        - name: ecs_service works fine even when older botocore is used
+          assert:
+            that:
+              - ecs_service_creation.changed
+
```

Comment:
You could remove a couple lines of extra whitespace.

Reply:
I have no problem with that; do you want another push with them removed?

Reply:
@ezmac  You'll need to remove the blank lines, then rebase to resolve the conflicts

https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

Reply:
New to this kind of review; requested changes made plus a few other extra blank lines.  Code has been rebased on current devel, so should be good.